apps/editing-toolkit/editing-toolkit-plugin/whats-new/src/whats-new.js

```diff
@@ -4,184 +4,36 @@ import './public-path';
 /**
  * External dependencies
  */
+import { useState } from 'react';
 import { registerPlugin } from '@wordpress/plugins';
-import { Fill, Guide, GuidePage, MenuItem } from '@wordpress/components';
-import { useDispatch, useSelect } from '@wordpress/data';
+import { Fill, MenuItem } from '@wordpress/components';
 import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { useEffect, createInterpolateElement } from '@wordpress/element';
+import WhatsNewGuide from '@automattic/whats-new';
```

Comment:
Minor nit: Ideally, we try to sort imports alphabetically. Rumors have it that it can make things more readable and scannable for some developers 😉